.github/CODEOWNERS

```diff
@@ -1,3 +1,19 @@
 # Tier Codeowners for framework based changes
 
 * @SatelliteQE/robottelo-tier-1-reviewers
+
+/* @SatelliteQE/robottelo-tier-2-reviewers
+
+/.github/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/conf/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/pytest_fixtures/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/pytest_plugins/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/robottelo/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/scripts/ @SatelliteQE/robottelo-tier-2-reviewers
+
+/tests/foreman/sanity/ @SatelliteQE/robottelo-tier-2-reviewers
```

Comment:
wdyt?
```suggestion
/tests/foreman/sanity/ @SatelliteQE/robottelo-tier-2-reviewers

/tests/robottelo/ @SatelliteQE/robottelo-tier-2-reviewers

/docs/ @SatelliteQE/robottelo-tier-2-reviewers
```

Reply:
I think docs are worth adding for the T2 review which I believe is being less touched last touch in that file has 2/3 years ago. We can get rid of docs if that's ACKed by all @SatelliteQE/airgun-tier-2-reviewers stakeholders. 

Rest, There is a less chance that anyone will touch `tests/robottelo` without changing anything in framework changes also tests inside `tests/robottelo` are part of CI and getting them passing is mandatory. So dont want to enforce T2 review there.